CUJExecutor/app/src/androidTest/java/com/example/firstapp/UserCujTest.java

```diff
@@ -42,4 +42,14 @@ public void runCuj() throws Exception {
         ShellUtility shellUtility = new ShellUtility(TIMEOUTMS);
         shellUtility.executeCUJ(preCUJ, postCUJ, iterations, recordIntent);
     }
+
+    @Test
+    public void dontRunLast() throws Exception {
```

Comment:
Instead of this new test method, what about passing in arguments to the existing one?

Reply:
that's what I originally did (commented out in yesProfileCuj), but the existing one has no way to "skip" the last action currently. Could extend it, but doesn't really apply to the canonical use case